### ISSUE TITLE
 Comparing Raw HTML vs `strip-tags` vs Markdown

### ISSUE THREAD

**Issue by benjamin-kirkbride** [simonw/strip-tags#11]
I am going to be doing some experiments with using raw html vs stripping tags vs converting to markdown against a few axis:
- token usage
- llm performance (not sure how to do this exactly)

Doing the above is what motivated me to do https://github.com/simonw/strip-tags/issues/9

This will solve https://github.com/simonw/strip-tags/issues/8

**Comment by benjamin-kirkbride** [comment]
Work taking place here: https://github.com/benjamin-kirkbride/llm-html-solutions

**Comment by simonw** [comment]
Definitely interested to see what you learn!

**Comment by simonw** [comment]
Related to this, I just shipped a new feature which allows you to selectively retain specific HTML elements - so you can keep the `<h1>...</h1>` tags and let the language model know which bits of the page were headers, for example.

I imagine this will be worth comparing with your Markdown work.

Details here:
- #14

**Comment by benjamin-kirkbride** [comment]
Yeah I think headings, tables, and links are the 3 big things, all of which (most) HTML to Markdown converters retain, so yeah definitely worth comparing

**Comment by benjamin-kirkbride** [comment]
also lists
